ckanext/s3filestore/views/resource.py

```diff
@@ -99,6 +101,19 @@ def resource_download(package_type, id, resource_id, filename=None):
         return redirect(rsc[u'url'])
 
 
+def s3_download(url):
+    s3_response = requests.get(url)  
+    response = flask.make_response(s3_response.content)
```

Comment:
Loading into CKAN memory is undesirable for very large files.

Reply:
Can you elaborate on that? Because we're using our fork of this plugin on CKAN 2.10 at www.data.qld.gov.au and www.publications.qld.gov.au, with redirect, and it's working fine.

Reply:
After migrating of the portal, we start getting this error
```
(redirected from 'http://localhost:5000/dataset/8e7835d3-c628-40e7-a3da-2f159e6c6db3/resource/fd7b1564-5d46-4869-af50-d12758b7dba6/download/capitan_reef___depth_to_water_capitan_reef_dtw.geojson') from origin 'http://localhost:5000/' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```